### DISCUSSION TITLE
lighting effects with normal led stripes

### DISCUSSION THREAD

**Discussion by Hanimaniac** [RavenSystem/esp-homekit-devices#1669] [Q&A]
Hey Friends,

is there a possibility for lighting effects with normal singlecolor -or rgb led strips, eg: fade in fade out or fade trough colors?
Or is it just possible with adressable leds?

Thanks Martin

**Comment by wahidulislamriyad** [comment]
Those LEDs has their own logic to change colors as far as I know. You need LEDs which has 4 pins for Red, Green, Blue and Ground.
